src/flanders/spec.clj

```diff
@@ -107,6 +107,32 @@
       (eval `(s/def ~result-kw ~(f type (str ns "." "set-of"))))
       (eval `(s/coll-of ~result-kw :kind set?))))
 
+  SignatureType
+  (->spec' [this ns f]
+    (let [parameter-list (get this :parameters)
+          parameters (get parameter-list :parameters)
+          rest-parameter (get this :rest-parameter)
```

Comment:
Why are you not using map de-structuring here? `{:keys [parameters rest-parameters] :as this}`
I would keep the the `get` for the second level. I don't like multi level de-structuring

Reply:
I'm not a fan of map destructing and have gotten out of the habit of using it, however, I have no problem using it if you find it preferable in this instance. 👍